### PR TITLE
LINK-1970 | update event price section instruction texts

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Update event price section instruction texts

## Closes
[LINK-1970](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1970)

## Screenshot
Screenshot 2024-06-05 at 10 45 55

[LINK-1970]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1970?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr406-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr406-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/app/i18n/en.json [hunk 1]
```diff
@@ -610,15 +610,20 @@
...
+        "general": "If you create a Linked Events registration for an event, enter the customer groups and their prices. This information will be transferred to the enrolment.",
+        "volunteering": "If you create a Linked Events registration for a volunteering, enter the customer groups and their prices. This information will be transferred to the enrolment."
       },
       "infoTextOffers2": {
         "course": "If the course has multiple prices, click “Add other pricing information” and enter the name of each price group (e.g. Children) in the “Additional price information” field.",
         "general": "If the event has multiple prices, click “Add other pricing information” and enter the name of each price group (e.g. Children) in the “Additional price information” field.",
         "volunteering": "If the volunteering has multiple prices, click “Add other pricing information” and enter the name of each price group (e.g. Children) in the “Additional price information” field."
       },
+      "infoTextOffers3": {
+        "course": "Jos luot kurssille Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```
Comment (harriris-vincit) [inline src/domain/app/i18n/en.json:623]:
```suggestion
        "course": "Jos luot kurssille Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```

File: src/domain/app/i18n/en.json [hunk 2]
```diff
@@ -610,15 +610,20 @@
...
+        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```
Comment (harriris-vincit) [inline src/domain/app/i18n/en.json:624]:
```suggestion
        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```

File: src/domain/app/i18n/en.json [hunk 3]
```diff
@@ -610,15 +610,20 @@
...
+        "volunteering": "Jos luot vapaaehtoistehtävälle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle."
```
Comment (harriris-vincit) [inline src/domain/app/i18n/en.json:625]:
```suggestion
        "volunteering": "Jos luot vapaaehtoistehtävälle Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle."
```

File: src/domain/app/i18n/fi.json [hunk 4]
```diff
@@ -610,15 +610,20 @@
...
+        "volunteering": "Hinta- ja hintatietojen kuvaus -kenttien tiedot menevät tapahtumailmoitukselle näkyviin. Syötä maksulliselle vapaaehtoistehtävälle hinta muodossa “5€”."
       },
       "infoTextOffers2": {
         "course": "Jos kurssilla on useita hintoja, klikkaa “Lisää muita hintatietoja” ja merkitse kunkin hintaryhmän nimi (esim. Lapset) Hintatietojen kuvaus -kenttään.",
         "general": "Jos tapahtumalla on useita hintoja, klikkaa “Lisää muita hintatietoja” ja merkitse kunkin hintaryhmän nimi (esim. Lapset) Hintatietojen kuvaus -kenttään.",
         "volunteering": "Jos vapaaehtoistehtävällä on useita hintoja, klikkaa “Lisää muita hintatietoja” ja merkitse kunkin hintaryhmän nimi (esim. Lapset) Hintatietojen kuvaus -kenttään."
       },
+      "infoTextOffers3": {
+        "course": "Jos luot kurssille Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
+        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```
Comment (harriris-vincit) [inline src/domain/app/i18n/fi.json:624]:
```suggestion
        "general": "Jos luot tapahtumalle Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle.",
```

File: src/domain/app/i18n/fi.json [hunk 5]
```diff
@@ -610,15 +610,20 @@
...
+        "volunteering": "Jos luot vapaaehtoistehtävälle Linked Events -ilmoittautumisen. syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle."
```
Comment (harriris-vincit) [inline src/domain/app/i18n/fi.json:625]:
```suggestion
        "volunteering": "Jos luot vapaaehtoistehtävälle Linked Events -ilmoittautumisen, syötä asiakasryhmät ja niille hinnat. Nämä tiedot siirtyvät ilmoittautumiselle."
```

File: src/domain/app/i18n/sv.json [hunk 6]
```diff
@@ -610,15 +610,20 @@
...
+        "volunteering": "Informationen i fälten pris och prisbeskrivning visas i evenemangets annons. Ange ett pris i formatet \"5 €\" för betald volontärverksamhet."
       },
       "infoTextOffers2": {
         "course": "Om kursen har flera priser klickar du på “Lägg till annan prisinformation” och anger namnet på varje prisgrupp (t.ex. barn) i fältet “Ytterligare prisinformation”.",
         "general": "Om evenemanget har flera priser klickar du på “Lägg till annan prisinformation” och anger namnet på varje prisgrupp (t.ex. barn) i fältet “Ytterligare prisinformation”.",
         "volunteering": "Om volontärarbetet har flera priser klickar du på “Lägg till annan prisinformation” och anger namnet på varje prisgrupp (t.ex. barn) i fältet “Ytterligare prisinformation”."
       },
+      "infoTextOffers3": {
+        "course": "Om du skapar en Linked Events-registrering för en kurs anger du kundgrupperna och deras priser. Denna information kommer att överföras till registreringen.",
+        "general": "Om du skapar en Linked Events-registrering för ett evenemang anger du kundgrupperna och deras priser. Denna information kommer att överföras till registreringen.",
```
Comment (harriris-vincit) [inline src/domain/app/i18n/sv.json:624]:
```suggestion
        "general": "Om du skapar en Linked Events-registrering för ett evenemang, ange kundgrupper och deras priser. Denna information kommer att överföras till registreringen.",
```

File: src/domain/app/i18n/sv.json [hunk 7]
```diff
@@ -610,15 +610,20 @@
...
+        "volunteering": "Om du skapar en Linked Events-registrering för ett volontärarbete anger du kundgrupperna och deras priser. Denna information kommer att överföras till registreringen."
```
Comment (harriris-vincit) [inline src/domain/app/i18n/sv.json:625]:
```suggestion
        "volunteering": "Om du skapar en Linked Events-registrering för ett volontärarbete, ange kundgrupper och deras priser.. Denna information kommer att överföras till registreringen."
```

**Comment by jorilindell** [comment]
@harriris-vincit I implemented all the translations improvement suggestion. At same time I noticed that there was some Finnish texts in the English translation file. I fixed them as well.

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr406-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr406-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**